frontend/src/main/utils/currentUser.js

```diff
@@ -20,7 +19,7 @@ export function useCurrentUser() {
       return { loggedIn: false, root: null };
     }
   }, {
-    initialData: { loggedIn: false, root: null }
+    initialData: { loggedIn: false, root: null, initialData:true }
```

Comment:
This was for a debugging strategy during test development to disambiguate between two occurences of values `{ loggedIn: false, root: null }`.   It can be removed now.